titles/admin-rhdh/title-admin.adoc

```diff
@@ -18,6 +18,9 @@ include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
 
 include::assemblies/assembly-enabling-authentication.adoc[leveloffset=+1]
 
+// Running RHDH behind a proxy
+include::modules/admin/proc-running-rhdh-behind-a-proxy.adoc[leveloffset=+1]
```

Comment:
In the future, I would organize this into an assembly with multiple proc modules. For now, this is a great start to making this content available to users.